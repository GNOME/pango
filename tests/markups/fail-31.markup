<span gravity-hint="blue">test</span>
