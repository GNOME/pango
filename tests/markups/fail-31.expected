ERROR: 'blue' is not a valid value for the 'gravity_hint' attribute on <span> tag, line 1; valid values are natural/strong/line